Sprint-1/destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,13 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+hogwarts.forEach(({firstName, lastName, house, }) =>{
+  if (house === "Gryffindor")
+  console.log(`${firstName} ${lastName}`);
+});
+
+hogwarts.forEach(({firstName, lastName, pet, occupation}) =>{
```

Comment:
nit: It will be helpful for the person reading the output if there is a text with context before outputting the list.
i.e. Teachers with pets: